### PR TITLE
beta - 4.67.0 - release notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 ## 4.67.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_**_iam_policy` ([#14662](https://github.com/hashicorp/terraform-provider-google/pull/14662))
```
Comment (melinath) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_*_iam_policy` ([#14662](https://github.com/hashicorp/terraform-provider-google/pull/14662))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+* **New Data Source:** google_vertex_ai_index ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
```
Comment (melinath) [inline CHANGELOG.md:4]:
```suggestion
* **New Data Source:** `google_vertex_ai_index` ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+
+IMPROVEMENTS:
+* cloudrun: Added beta field `template.spec.columes.empty_dir` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```
Comment (melinath) [inline CHANGELOG.md:7]:
```suggestion
* cloudrun: Added field `template.spec.volumes.empty_dir` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+* cloudrun: Added field `template.spec.containers.name` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```
Comment (melinath) [inline CHANGELOG.md:8]: it looks like all of the links are to tpg instead of tpg-beta

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+* compute: - promoted field `network_performance_config` to ga ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
```
Comment (melinath) [inline CHANGELOG.md:9]: Note that in GA this should probably be rewritten to say something like "added field X to Y"

```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,24 @@
...
+* compute: added `guest_os_features` and `licenses` fields to `google_compute_disk` and `google_compute_region_disk` ([#14660](https://github.com/hashicorp/terraform-provider-google/pull/14660))
+* datastream: added `mysql_source_config.max_concurrent_backfill_tasks` field to `google_datastream_stream` resource ([#14639](https://github.com/hashicorp/terraform-provider-google/pull/14639))
+* firebase: added additional import formats for app resources, removed the need to supply duplicate projects in some cases ([#14638](https://github.com/hashicorp/terraform-provider-google/pull/14638))
```
Comment (melinath) [inline CHANGELOG.md:12]: this should possibly be rewritten to emphasize user impact / not use as much technical language

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,24 @@
...
+* notebooks: added update-in-place support for `google_notebooks_instance.metadata` field ([#14650](https://github.com/hashicorp/terraform-provider-google/pull/14650))
+* privateca: added support for encoding_format on CaPool ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
```
Comment (melinath) [inline CHANGELOG.md:14]:
```suggestion
* privateca: added support for `encoding_format` on `google_privateca_ca_pool` ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,24 @@
...
 
+BUG FIXES:
+* apigee: increased `google_apigee_organization` timeout defaults to 45m from 20m ([#14643](https://github.com/hashicorp/terraform-provider-google/pull/14643))
+* cloudrun: fixed an issues where the system annotation "run.googleapis.com/operation-id" and "run.googleapis.com/ingress" may cause permadiffs for `metadata.annotation` in `google_cloud_run_service` ([#14642](https://github.com/hashicorp/terraform-provider-google/pull/14642))
+* container: fixed a crash when all zero values are specified `google_container_node_pool.upgrade_settings.blue_green_settings.standard_rollout_policy` subfields ([#14693](https://github.com/hashicorp/terraform-provider-google/pull/14693))
+* gkeonprem: updated `hostname` field of ip_block from required to optional from resource `google_gkeonprem_vmware_cluster` ([#14690](https://github.com/hashicorp/terraform-provider-google/pull/14690))
+* provider: fixed an issue where mtls transports were not used consistently ([#14621](https://github.com/hashicorp/terraform-provider-google/pull/14621))
```
Comment (melinath) [inline CHANGELOG.md:21]: should probably be rewritten to emphasize user impact

**Review by melinath — CHANGES_REQUESTED** [review]
it looks like all the links are for the GA provider rather than beta - is there any chance that this was generated from the GA provider? If so it might be good to re-generate from the beta provider to make sure nothing is missed. Other comments left inline :-)

**Review by roaks3 — APPROVED** [review]

**Pull request merged**